client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityRequestConverters.java

```diff
@@ -174,6 +175,15 @@ static Request invalidateToken(InvalidateTokenRequest invalidateTokenRequest) th
         return request;
     }
 
+    static Request getPrivileges(GetPrivilegesRequest getPrivilegesRequest) throws IOException {
+        String endpoint = new RequestConverters.EndpointBuilder()
+            .addPathPartAsIs("_xpack/security/privilege")
+            .addPathPart(getPrivilegesRequest.getApplicationName())
+            .addPathPart(getPrivilegesRequest.getPrivilegeName())
```

Comment:
Should be `addCommaSeparatedPathParts` .

Reply:
Good catch, I missed that we support requests to get multiple privileges of an application. Will address